VolokhovAnton/HW7/7.3.py

```diff
@@ -0,0 +1,25 @@
+def count_characters(input_string):
```

Comment:
good job
look at this code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result